examples/flutter_view/ios/Runner/MainViewController.m

```diff
@@ -36,13 +33,22 @@ - (void)prepareForSegue:(UIStoryboardSegue*)segue sender:(id)sender {
   }
 
   if ([segue.identifier isEqualToString:@"FlutterViewControllerSegue"]) {
-     self.flutterViewController = segue.destinationViewController;
-    [self.flutterViewController addMessageListener:self];
+    self.flutterViewController = segue.destinationViewController;
+
+    self.messageChannel = [FlutterMessageChannel messageChannelNamed:channel
```

Comment:
This is the earliest point in time where we have the FlutterViewController which is needed when the channel is initialized so I believe it should stay here.